src/main/resources/db/migration/V0001__init.sql

```diff
@@ -0,0 +1,481 @@
+------------------------------------------------
+CREATE INDEX "BCA_BULK_CUSTOMER_I" ON "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS" ("BULK_CUSTOMER_ID");
+CREATE INDEX "BCA_TARGET_APPLICATION_I" ON "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS" ("TARGET_APPLICATION_ID");
+CREATE INDEX "BS_BULK_CUSTOMER_ID_I" ON "SDT_OWNER"."BULK_SUBMISSIONS" ("BULK_CUSTOMER_ID");
+CREATE INDEX "BS_SDT_BULK_REFERENCE_I" ON "SDT_OWNER"."BULK_SUBMISSIONS" ("SDT_BULK_REFERENCE");
+CREATE INDEX "BS_SERVICE_REQUEST_ID_I" ON "SDT_OWNER"."BULK_SUBMISSIONS" ("SERVICE_REQUEST_ID");
+CREATE INDEX "BS_TARGET_APPLICATION_ID_I" ON "SDT_OWNER"."BULK_SUBMISSIONS" ("TARGET_APPLICATION_ID");
+CREATE UNIQUE INDEX "BULK_CUSTOMERS_PK" ON "SDT_OWNER"."BULK_CUSTOMERS" ("BULK_CUSTOMER_ID");
+CREATE UNIQUE INDEX "BULK_CUSTOMER_APPLICATIONS_PK" ON "SDT_OWNER"."BULK_CUSTOMER_APPLICATIONS" ("BULK_CUSTOMER_APPLICATIONS_ID");
+CREATE UNIQUE INDEX "BULK_SUBMISSIONS_PK" ON "SDT_OWNER"."BULK_SUBMISSIONS" ("BULK_SUBMISSION_ID");
+CREATE INDEX "EL_INDIVIDUAL_REQUEST_ID_I" ON "SDT_OWNER"."ERROR_LOGS" ("INDIVIDUAL_REQUEST_ID");
+CREATE UNIQUE INDEX "ERROR_LOGS_PK" ON "SDT_OWNER"."ERROR_LOGS" ("ERROR_LOG_ID");
+CREATE UNIQUE INDEX "ERROR_MESSAGES_PK" ON "SDT_OWNER"."ERROR_MESSAGES" ("ERROR_MESSAGE_ID");
+CREATE UNIQUE INDEX "GLOBAL_PARAMETERS_PK" ON "SDT_OWNER"."GLOBAL_PARAMETERS" ("GLOBAL_PARAMETER_ID");
+CREATE UNIQUE INDEX "INDIVIDUAL_REQUESTS_PK" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("INDIVIDUAL_REQUEST_ID");
+CREATE INDEX "IR_BULK_REFERENCE_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("SDT_BULK_REFERENCE");
+CREATE INDEX "IR_BULK_SUBMISSION_ID_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("BULK_SUBMISSION_ID");
+CREATE INDEX "IR_LWR_CUSTOMER_REQ_REF_CD_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" (LOWER("CUSTOMER_REQUEST_REF"), "CREATED_DATE");
+CREATE INDEX "IR_SDT_REQUEST_REFERENCE_I" ON "SDT_OWNER"."INDIVIDUAL_REQUESTS" ("SDT_REQUEST_REFERENCE");
+CREATE UNIQUE INDEX "SERVICE_REQUESTS_PK" ON "SDT_OWNER"."SERVICE_REQUESTS" ("SERVICE_REQUEST_ID");
+CREATE UNIQUE INDEX "SERVICE_ROUTINGS_PK" ON "SDT_OWNER"."SERVICE_ROUTINGS" ("SERVICE_ROUTINGS_ID");
+CREATE UNIQUE INDEX "SERVICE_TYPES_PK" ON "SDT_OWNER"."SERVICE_TYPES" ("SERVICE_TYPE_ID");
+CREATE INDEX "SR_SERVICE_TYPE_I" ON "SDT_OWNER"."SERVICE_ROUTINGS" ("SERVICE_TYPE_ID");
+CREATE INDEX "SR_TARGET_APPLICATION_I" ON "SDT_OWNER"."SERVICE_ROUTINGS" ("TARGET_APPLICATION_ID");
+CREATE INDEX "ST_SERVICE_TYPE_NAME" ON "SDT_OWNER"."SERVICE_TYPES" ("SERVICE_TYPE_NAME");
+CREATE UNIQUE INDEX "TARGET_APPLICATIONS_PK" ON "SDT_OWNER"."TARGET_APPLICATIONS" ("TARGET_APPLICATION_ID");
+CREATE INDEX "TA_TARGET_APPLICATION_NAME" ON "SDT_OWNER"."TARGET_APPLICATIONS" ("TARGET_APPLICATION_NAME");
+
+------------------------------------------------
+-- Create Primary Keys for SDT_OWNER
+------------------------------------------------
+ALTER TABLE "SDT_OWNER"."BULK_CUSTOMERS" ADD CONSTRAINT "BULK_CUSTOMERS_PK" PRIMARY KEY USING INDEX "BULK_CUSTOMERS_PK";
```

Comment:
Do indexes need to be prefixed with owner (e.g. "SDT_OWNER"."BULK_CUSTOMERS_PK")?

Reply:
No. Doesn't like that syntax.